### ISSUE TITLE
Prevent BERT overfitting with small dataset

### ISSUE THREAD

**Issue by svenvanderburg** [SSIML2021/SSIML2021#36]
Our end-to-end BERT model for causal presence detection is performing quite bad, I expect more from it. The model is now very heavily overfitting to the training data.
* Try a much smaller learning rate
* Try out ways of preventing overfitting of BERT model on small dataset (which we have) that were published in [this paper](https://openreview.net/pdf?id=cO1IH43yUF)
